presto-product-tests-utils/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>presto-root</artifactId>
+        <groupId>com.facebook.presto</groupId>
+        <version>0.116-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-product-tests-utils</artifactId>
+    <name>presto-product-tests-utils</name>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-main</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-jdk14</artifactId>
```

Comment:
Why is this being excluded? The presto codebase assumes that everything logs to java.util.logging. We need to be very careful with how logging is wired up, or it's easy to end up in dependency hell, missed logging, double logging, etc. Here's an explanation of how logging works in presto: https://groups.google.com/d/msg/presto-users/2hN1ZGw-SVA/jyyhf_H3SjcJ. Some details have changed since I wrote that, but the general idea stands.